packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -48,10 +50,12 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
         { ...{ onBlurCapture: this._onTooltipMouseLeave } }
         onMouseEnter={ this._onTooltipMouseEnter }
         onMouseLeave={ this._onTooltipMouseLeave }
+        aria-describedby={ tooltipId }
```

Comment:
Should we add aria-describedby only when tooltip is visible? otherwise it will point to an invalid element.